CHANGELOG.md

```diff
@@ -4,109 +4,110 @@
+- Removed deprecated `getActiveDomain` method and `DomainAsCarrier` type from `@sentry/hub`. (#4858)
+- Removed `eventStatusFromHttpCode` to save on bundle size.
+- Removed usage of deprecated `event.stacktrace` field. (#4885)
+- Removed raven-node backward-compat code (#4942)
+- Removed `showReportDialog` method on `BrowserClient` (#4973)
+- Removed deprecated `startSpan` and `child` methods (#4849)
+- Removed deprecated `frameContextLines` options (#4884)
+- Removed `Sentry` from window in the Gatsby SDK (#4857)
+
+#### Internal
+
+- Removed support for the store endpoint (#4969)
+- Made hint callback argument non-optional (#5141)
+- Switched to using new transports internally (#4943)
+- [Removed `API` class from `@sentry/core`.](./MIGRATION.md#removing-the-api-class-from-sentrycore). (#4848)
+- [Refactored `Session` class to use a more functional approach.](./MIGRATION.md#session-changes). (#5054)
+- Removed `Backend` class in favour of moving functionality into the `Client` class (for more details, see [#4911](https://github.com/getsentry/sentry-javascript/pull/4911) and [#4919](https://github.com/getsentry/sentry-javascript/pull/4919)).
+- Removed forget async utility function (#4941)
+- Removed tslint from `@sentry-internal/typescript` (#4940)
+- Removed `_invokeClient` function from `@sentry/hub` (#4972)
+- Removed top level eventbuilder exports (#4887)
+
+#### Features
 
-- **(breaking)** feat(nextjs): Make tracing package treeshakable (#5166)
-- fix(browser): Fix memory leak in `addEventListener` instrumentation (#5147)
-- fix(tracing): Don't use `querySelector` when not available (#5160)
-- fix(utils): Fix infinite recursion in `dropUndefinedKeys` (#5163)
-- ref(build): Use rollup to build AWS lambda layer (#5146)
-- ref(core): Actually ensure attachments are added to the envelope (#5159)
-- ref(core): Make hint callback argument non-optional (#5141)
-
-## 7.0.0-beta.2
-
-- **(breaking)** feat(tracing): Add empty baggage header propagation to outgoing requests (#5133)
+- feat(tracing): Add Prisma ORM integration. (#4931)
+- feat(react): Add react-router-v6 integration (#5042)
 - feat: Add attachments API (#5004)
-- feat(core): Send Baggage in Envelope Header (#5104)
-- feat(utils): Introduce Baggage API (#5066)
-- fix(build): Fix express import in `gcpfunction` (#5097)
-- fix(ember): Export sha hashes of injected scripts (#5089)
-- ref(build): Use sucrase for es6 bundles (#5111)
-- ref(serverless): Do not throw on flush error (#5090)
-- ref(serverless): Point DSN to relay in lambda extension (#5126)
-- ref(tracing): Rename baggage env header to trace (#5128)
 
-## 7.0.0-beta.1
+- feat: Add `name` field to `EventProcessor` (#4932)
+- feat: Expose configurable stack parser (#4902)
+- feat(tracing): Make `setMeasurement` public API (#4933)
+- feat(tracing): Add GB unit to device memory tag value (#4935)
+- feat: Export browser integrations individually (#5028)
+- feat(utils): Introduce Baggage API (#5066)
```

Comment:
Wondering if we should move this to internal

Reply:
Yup agree, will move!